src/main/webapp/features/screens/game/dashboard/ct-game-dashboard.html

```diff
@@ -0,0 +1,3 @@
+game dashboard
+<br><br>
+object "game" is available here
```

Comment:
it would be good to have some stub here, that represents game map overview, and details of the game in text format, jsut to be sure, that game is really avaliable here